src/main/java/com/avispl/symphony/dal/communicator/barco/e2gen2/BarcoE2Communicator.java

```diff
@@ -1281,11 +1452,16 @@ private void getSuperRoutingControl(boolean isSuperDest, Map<String, String> sta
 		List<String> sourceList = new ArrayList<>(sourceIdToNameMap.values());
 		// Super Screen/Aux dest loop
 		for (Integer listDestId : listDestIds) {
-			JsonNode response;
-			response = getSuperDestContent(isSuperDest, String.valueOf(listDestId));
+			JsonNode response = getSuperDestContent(isSuperDest, String.valueOf(listDestId));
 			if (response == null) {
 				return;
 			}
+			if (response.get(BarcoE2Constant.SUCCESS_STATUS) != null && response.get(BarcoE2Constant.SUCCESS_STATUS).asInt() == -1) {
```

Comment:
response.get(BarcoE2Constant.SUCCESS_STATUS) can be extracted and reused

Reply:
Fixed